apps/passport-server/package.json

```diff
@@ -36,6 +36,8 @@
     "@pcd/zk-eddsa-ticket-pcd": "0.1.0",
     "@semaphore-protocol/group": "^3.11.0",
     "@semaphore-protocol/identity": "^3.11.0",
+    "@semaphore-protocol/proof": "^3.11.0",
```

Comment:
I don't see where `@semaphore-protocol/proof`or `@sendgrid/mail` is used in the `passport-server`